build-system/tasks/dep-check.js

```diff
@@ -311,7 +310,14 @@ async function depCheck() {
     .then(runRules)
     .then(errorsFound => {
       if (errorsFound) {
-        process.exit(1);
+        log(
+          yellow('NOTE:'),
+          'If a dependency is valid, add it to one of the whitelists in',
+          cyan('build-system/dep-check-config.js')
+        );
+        const reason = new Error('Dependency checks failed');
```

Comment:
nit: outputting this after the task failed is a little redundant because the error logging is so clear now

Reply:
This is to signal completion to the `gulp` task runner. Without it, you'll see this:

![image](https://user-images.githubusercontent.com/26553114/62141416-8960e280-b2ba-11e9-835f-573db9355a96.png)

Reply:
not ok to just return `Promise.reject()`?